cmd/kube-apiserver/app/options/validation.go

```diff
@@ -23,12 +23,11 @@ import (
 // TODO: Longer term we should read this from some config store, rather than a flag.
 func validateClusterIPFlags(options *ServerRunOptions) []error {
 	errors := []error{}
-	if options.ServiceClusterIPRange.IP == nil {
```

Comment:
This validation still looks correct.  The options could be embedded somewhere else (its why `kubectl` built the options structs to beging with), so the validation should remain.